app/src/test/java/org/simple/clinic/registration/name/RegistrationFullNameScreenControllerTest.kt

```diff
@@ -88,9 +101,12 @@ class RegistrationFullNameScreenControllerTest {
     uiEvents.onNext(RegistrationFullNameTextChanged(""))
     uiEvents.onNext(RegistrationFullNameDoneClicked())
 
-    verify(screen).showEmptyNameValidationError()
     verify(userSession, never()).saveOngoingRegistrationEntry(any())
```

Comment:
This implementation will be tested by `verifyNoMoreInteractions`, doesn't it?

Reply:
It was actually moved so that all the assertions on `screen` are grouped together.

Reply:
https://github.com/simpledotorg/simple-android/pull/1411/commits/741ff6b58d274ec721c6d1f19e7d5b9ddf082a38#diff-1c99d8956d8cb6d463b776cd5a65dca6R107

Reply:
Oh, I was asking about `verify(userSession, never()).saveOngoingRegistrationEntry(any())`, not sure why GH pointed to above method.

Reply:
Oh, it won't be tested since `verifyNoMoreInteractions` is not taking `userSession` as a parameter.

> not sure why GH pointed to above method.

It was. I just think my eyes were drawn to the line above since it was highlighted as part of the diff. 😓

Reply:
Ah right, I missed it. 😅